src/execution/window_executor.cpp

```diff
@@ -643,6 +799,10 @@ bool WindowAggregateExecutor::IsConstantAggregate() {
 	if (!wexpr.aggregate) {
 		return false;
 	}
+	// window exclusion cannot be handled by constant aggregates
+	if (wexpr.exclude_clause != WindowExclusion::NO_OTHER) {
+		return false;
+	}
 
 	//	COUNT(*) is already handled efficiently by segment trees.
```

Comment:
Sorry this comment is not correct (it used to be before I refactored!)  `COUNT(*)` is now the easiest example of a custom window function.